docs/configuration.md

```diff
@@ -80,6 +115,41 @@ sentinels.
 - Applies to: listener, worker, cron.
 - Format: string.
 
+###  CONFIG_QUEUES_USERNAME
```

Comment:
This I guess is for the `resque` background jobs. So I suggest naming it `CONFIG_RESQUE_USERNAME` or `CONFIG_JOBS_USERNAME`. This `QUEUES` word is a little ambiguous.

btw how nice that we have yet another background jobs implementation to care about...

Reply:
The `CONFIG_QUEUES_*` prefix was already there before, I prefer to not touch it.

Reply:
hmm, I wondered how did you come up with this name.. I don't like it but what you say makes sense.